third-test.md

```diff
@@ -0,0 +1,29 @@
+# Third Test Pattern
+
+## Title
+
+_Third Test_
+
+## Problem
+
+Still none.
+
+## Context
+
+Dunno.
+
+## Forces
+
+Slight pull to the light side.
+
+## Solution
+
+Similar to _insert pattern link to [First Test](first-test.md)_
```

Comment:
Links only work for pattern files wich are already on `master`. At least that's what I found yesterday. Maybe I'm doing it wrong ...